### DISCUSSION TITLE
Normalized IDs for winget

### DISCUSSION THREAD

**Discussion by Ivanv0** [Devolutions/UniGetUI#3106] [General]
I wanted to add some icons in google sheets. First I checked it with custom database, but I faced with the fact that some winget packages have the same normalized id.

For example:
1. 0xGingi.Browser
2. Dichromate.Browser
3. Yandex.Browser

at the moment, these winget packages have the same normalized id.
The same thing happens with other packages with simple names like music.

Is there any way to distinguish them?

**Comment by marticliment** [comment]
Not really, but if said packages already provide an icon via the package manager (which they are likely to) the custom icon won't show.

This is a thing I have to fix
